client/src/components/SudoModePasswordField/SudoModePasswordField.js

```diff
@@ -15,14 +15,41 @@ function SudoModePasswordField(props) {
   const {
     onSuccess,
     autocomplete,
+    collapsed,
```

Comment:
Rename the prop to "initiallyCollapsed" - otherwise `collapsed` and `expanded` can both be true at the same time, which is wrong.

Reply:
Updated